testing/functional_placeholders_bitwise.cu

```diff
@@ -37,11 +37,11 @@ template<typename Vector> \
 { \
   void operator()(const size_t) \
   { \
-    static const size_t num_samples = 10000; \
-    const size_t zero = 0; \
+    constexpr size_t NUM_SAMPLES = 10000; \
+    constexpr size_t ZERO = 0; \
```

Comment:
Why the all caps names?

While we're fixing issues here, `std::size_t` over `size_t` would be nice.

Reply:
`num_samples` needed a new name because it shadows another variable. Since it's a compile-time constant I used caps since that's a common style for constants.